packages/calling/calling/versionPlugin.js

```diff
@@ -27,6 +30,7 @@ const replaceVersion = (err, data) => {
 };
 
 /**
+ * .
```

Comment:
Would be better to add some descriptions, now that we are public

Reply:
done